lib/client/commands/login.go

```diff
@@ -116,5 +140,24 @@ func (l *Login) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+        // TODO(isaac): wrap below in a function and call via defer
+        // will that still execute if the user ctrl-c's ?
+        grpclog.SetLoggerV2(grpclog.NewLoggerV2(os.Stdout, os.Stderr, os.Stderr))
+        grpc.EnableTracing = true
+        conn, err = grpc.Dial("localhost:8981", grpc.WithInsecure(), grpc.WithUnaryInterceptor(tokenAuthInterceptor(enCreds.Token)))
```

Comment:
Hmm, didn't think about this, but this port will always need to be assumed `8981` right? Since we aren't going to add flags for this?

Reply:
This occured to me as well.

We're creating an implicit coupling between enkit login and bbclientd...

I thought about making it so that this value comes from a build time variable - and then we update the bb_clientd config to also use that variable (so they stay in sync).

Probably worthwhile since I can't say for certain when we'll remove this in favor of setting up credential helper support.

Reply:
It turns out we have mechanisms to push overrides to the default flag values (without rebuilding/deploying enkit)

So if we make this a cobra flag we can change it if we change the address the daemon runs at.